modules/gapi/src/compiler/gcompiler.cpp

```diff
@@ -417,9 +423,10 @@ cv::GCompiled cv::gimpl::GCompiler::produceCompiled(GPtr &&pg)
 
 cv::GStreamingCompiled cv::gimpl::GCompiler::produceStreamingCompiled(GPtr &&pg)
 {
+    std::unique_ptr<GStreamingExecutor> pE(new GStreamingExecutor(std::move(pg), m_args, m_metas));
```

Comment:
What's the reason to pass `m_metas` in constructor? Isn't it passed down in `setup()`?

Reply:
Isn't your `m_metas` vector empty at this point?